portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,45 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Somto's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Somto Ejinkonye's Portfolio</h1>
+      <p>This is my portfolio.</p>      
+      <p>Hello, I am Somto Ejinkonye and I am a Junior studying Computer Engineering and NYU. I am passonate about cybersecurity, prototyping and embedded systems.</p>
```

Comment:
Google's HTML style guide suggests wrapping long lines to make it easier to read: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

You might consider adding some linebreaks for these long lines?

Reply:
Thanks for the heads up. I added line breaks.